src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/VariantsToTable.java

```
@@ -28,60 +28,78 @@
  *
  * <h3>Output</h3>
  * <p>
- * A tab-delimited file containing the values of the requested fields in the VCF file.
+ *     A tab-delimited file containing the values of the requested fields in the VCF file.
  * </p>
  *
  * <h3>Usage example</h3>
  * <pre>
- *     ./gatk-launch VariantsToTable \
+ *     gatk VariantsToTable \
  *     -V input.vcf \
- *     -F CHROM -F POS -F ID -F QUAL -F AC \
+ *     -F CHROM -F POS -F TYPE -GF AD \
  *     -O output.table
  * </pre>
  * <p>would produce a file that looks like:</p>
  * <pre>
- *     CHROM    POS ID      QUAL    AC
- *     1        10  .       50      1
- *     1        20  rs10    99      10
+ *     CHROM  POS        TYPE   HSCX1010N.AD  HSCX1010T.AD
+ *     1      31782997   SNP    77,0          53,4
+ *     1      40125052   SNP    97,0          92,7
+ *     1      65068538   SNP    49,0          35,4
+ *     1      111146235  SNP    69,1          77,4
  * </pre>
  *
- * <h3>Caveats</h3>
+ * <h3>Notes</h3>
  * <ul>
- *     <li>Some annotations cannot be applied to all variant sites, so VCFs typically contain records where some annotation values are missing. By default this tool will emit the special value NA for the missing annotations if you request export of an annotation for which not all records have values. You can override this behavior by setting --errorIfMissingData in the command line. As a result, the tool will throw an error if a record is missing a value.</li>
- *     <li>When you request export of FORMAT/sample-level annotations (such as GT), the annotations will be identified per-sample. If multiple samples are present in the VCF, the columns will be ordered alphabetically by sample name (SM tag).</li>
+ *     <li> It is common that certain annotations are absent for some variants. By default this tool will emit NA for an annotation that's missing. If you wish that the tool fail when it encounters a missing annotation, turn on the --error-if-missing-data flag in the command line. </li>
```

--> It is common for certain annotations to be absent for some variants. By default, this tool will emit an NA for such missing annotations. If you would rather fail a run upon encountering a missing annotation, then specify the --error-if-missing-data option.

---

done